### PR TITLE
THREESCALE-11154 doc update - new optional info params in tenant CR

### PR DESCRIPTION
**Pull request by valerymo**
Jira: https://issues.redhat.com/browse/THREESCALE-11154

- Documentation update  
- related to https://issues.redhat.com/browse/THREESCALE-10525

* Optional parameters added to Tenant CR:
  * fromEmail
  * supportEmail
  * financeSupportEmail
  * siteAccessCode

### CODE REVIEW

File: doc/operator-application-capabilities.md [hunk 1]
```diff
@@ -1278,6 +1278,34 @@ spec:
     name: tenant-secret
 ```
 
+You can set the following optional parameters in your {ProductShortName} Tenant via the `Tenant` CR:
```
Comment (MStokluska) [inline doc/operator-application-capabilities.md:1281]: "{ProductShortName}" ?

File: doc/tenant-reference.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ Generated using [github-markdown-toc](https://github.com/ekalinin/github-markdow
 | Master Account Credentials Secret | `masterCredentialsRef` | object | See [Master Secret](#Master-Secret) for more details | Yes |
 | Admin Secret | `passwordCredentialsRef` | object | See [Admin Secret](#Admin-Secret) for more details | Yes |
 | Tenant Credentials Secret | `tenantSecretRef` | object | See [Tenant Secret](#Tenant-Secret) for more details | No |
+| Finance Support Email             | `financeSupportEmail` | string | Finance support email address              | No           |
```
Comment (MStokluska) [inline doc/tenant-reference.md:32]:
```suggestion
| Finance Support Email             | `financeSupportEmail` | string | Finance support email address              | No          |
```

File: doc/operator-application-capabilities.md (hunk 1 above)
Comment (valerymo) [reply doc/operator-application-capabilities.md]: oh, sorry, copied ...
Updated. Thank you

File: doc/tenant-reference.md (hunk 2 above)
Comment (valerymo) [reply doc/tenant-reference.md]: Updated , aligned with existed ..
Thank you

**Review by MStokluska — APPROVED** [review]

**Pull request merged**